src/components/link-preview/embed.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
```

Comment:
i think we can switch this file name to `index`. the only reason we had the alternate `embed` naming was to make it easier to transition from the old style of link previews to the new version.